src/ontology/ro-edit.owl

```diff
@@ -6127,6 +6131,21 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
 SubObjectPropertyOf(obo:RO_0015003 obo:BFO_0000050)
 TransitiveObjectProperty(obo:RO_0015003)
 
+# Object Property: obo:RO_0016001 (disease or disorder)
```

Comment:
not a relation name.. shouldnt the name be something like `has phenotype or disease` or some such?

Reply:
Sorry, I think I was a little confused at your instructions then. Should I not have made a parent class for 'has disease' and 'has phenotype'? Should they just both be siblings under the top class?

Reply:
They are not classes, :) no I think just rename this object property to "has phenotype or disease" or whatever Chris suggested in the ticket is fine. But they are not classes!!

Reply:
Ok, not classes!! I relabeled the object property and added a note not to use this class directly. I think that's appropriate here, correct? @matentzn